libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -307,3 +312,32 @@ describe('ListComponent', () => {
     });
   });
 });
+
+describe('ListComponent with items declared in markup', () => {
+  let spectator: SpectatorHost<ListComponent>;
+
+  const createHost = createHostFactory({
+    component: ListComponent,
+    declarations: [ListComponent, MockComponent(ionic.IonList), MockComponent(ionic.IonItem)],
+  });
+
+  beforeEach(() => {
+    spectator = createHost<ListComponent>(`
+      <kirby-list>
+        <kirby-item><h3>Test Item</h3></kirby-item>
+        <kirby-item><h3>Test Item</h3></kirby-item>
+        <kirby-item><h3>Test Item</h3></kirby-item>
+        <kirby-item><h3>Test Item</h3></kirby-item>
+      </kirby-list>`);
```

Comment:
```suggestion
      <kirby-list>
        <kirby-item>Test Item</kirby-item>
        <kirby-item>Test Item</kirby-item>
        <kirby-item>Test Item</kirby-item>
        <kirby-item>Test Item</kirby-item>
      </kirby-list>`);
```

Just to keep it simple(r)